api/emails/orderSpeakerNotification.js

```diff
@@ -0,0 +1,43 @@
+export default data => {
+  const to = {
+    name: 'Ralph Caspers',
+    email: 'astrid@tactile.news'
+  }
+
+  const from = {
+    name: 'Franziska von tactile.news',
+    email: 'studio@tactile.news'
+  }
```

Comment:
@tillprochaska any ideas where to put that data later on?

Reply:
Actually, I thinks it’s fine to have the sender address in the same location as the email itself…